python/ray/serve/_private/constants.py

```diff
@@ -191,3 +191,9 @@ class ServeHandleType(str, Enum):
 
 # Serve HTTP request header key for routing requests.
 SERVE_MULTIPLEXED_MODEL_ID = "serve_multiplexed_model_id"
+
+# Feature flag to enable StreamingResponse support.
+# When turned on, *all* HTTP responses will use Ray streaming object refs.
+RAY_SERVE_ENABLE_EXPERIMENTAL_STREAMING = (
+    os.environ.get("RAY_SERVE_ENABLE_EXPERIMENTAL_STREAMING", "0") != "0"
```

Comment:
nitpick: the [doc](https://github.com/ray-project/ray/pull/35720/files#diff-d68d13b13572527636227edd9d7686c805ca021987431237d223b02dc2262e94R81) suggested setting `RAY_SERVE_ENABLE_EXPERIMENTAL_STREAMING` to 1 to enable this feature, but the code seems to be enabled as long as it's not 0. So setting it to 2, 3, True, False will all be enabled

Reply:
yeah let me update :)